src/widgets/tasks.rs

```diff
@@ -28,7 +28,7 @@ impl Tasks {
 
             let mut sorted_tasks = tasks.to_owned();
             sorted_tasks.sort();
-            sorted_tasks.reverse();
+            //sorted_tasks.reverse();
```

Comment:
Since the code is versioned, it'd be cleaner to just remove this LOC instead of commenting it out

Reply:
You are absolutely right :)